WORKSPACE

```diff
@@ -39,6 +39,11 @@ maven_jar(
     artifact = "com.google.truth:truth:jar:0.31",
 )
 
+maven_jar(
+    name = "com_spotify",
```

Comment:
Please use a canonical name since other dependencies from that groupId might be added later.
Something like `com_spotify_hamcrest_optional`